soc/bflb/bl60x/Kconfig.soc

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2021-2025 ATL Electronics
```

Comment:
Is it normal this file is marked as added? Did the previous file not get removed somehow?

Reply:
Yea i checked the the original file makes a return in the second commit.

Reply:
Thanks for paying attention! Still trying to get my head around how that happened, but should be good now, please check again